src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -647,12 +660,287 @@ void stream_generated_tokens(std::shared_ptr<ov::genai::StreamerBase> streamer_p
     }
 }
 
+enum PipelineKind {
+    STATEFUL,
+    STATELESS
+};
+PipelineKind str_to_pipeline(const std::string& str) {
+    if (str == "STATEFUL") {
+        return PipelineKind::STATEFUL;
+    }
+    if (str == "STATELESS") {
+        return PipelineKind::STATELESS;
+    }
+    OPENVINO_THROW("Unsupported \"PIPELINE\" provided: " +
```

Comment:
```suggestion
    OPENVINO_THROW("Unsupported \"PIPELINE\" provided: ", 
```
use commas instead of concatenation

Reply:
Fixed